lib/service.js

```diff
@@ -93,26 +88,31 @@ class Service {
     });
   }
 
+  static _checFormData(formData) {
+    if(isEmpty(formData)) {
+      throw new Error('Can\'t be empty');
+    }
+
+    if (!isObject(formData)) {
+      throw new Error('Should be an object');
+    }
+  }
```

Comment:
Iterate through the formData's values and validation each of them to make sure they have proper values as well.